sdk/identity/identity/src/credentials/managedIdentityCredential/imdsMsi.ts

```diff
@@ -78,7 +78,7 @@ function prepareRequestOptions(
 
 // 800ms -> 1600ms -> 3200ms
 export const imdsMsiRetryConfig = {
-  maxRetries: 3,
+  maxRetries: 5,
   startDelayInMs: 800,
   intervalIncrement: 2,
```

Comment:
What does this value do?

Reply:
It increases the no of max retries to 5

Reply:
Removed these changes to this PR - https://github.com/Azure/azure-sdk-for-js/pull/29139